### ISSUE TITLE
Website issue related to GD library

### ISSUE THREAD

**Issue by DominiqueVial** [fabianmichael/kirby-meta#53]
Right after installing this module I noticed problem on pages I used it on.

At first load site crashed. Examining debug I noticed the following error: 
"Required extension GD is not loaded."

![Capture d’écran 2023-07-31 à 16 46 19](https://github.com/fabianmichael/kirby-meta/assets/43033637/83bca108-cd00-413a-a9e3-27f8ba64df7d)

### Context

kirby-meta : 0.4.0-beta (up to date at the time this issue was created)
Kirby 3.9.6
PHP : 8.1.18
Server: Apache

**Comment by fabianmichael** [comment]
@DominiqueVial Hi, this is a problem of your webserver, since the PHP version installed was seemingly compiled without the GD library, which Kirby uses for image processing by default. You can find out more about Kirby’s requirements here: https://getkirby.com/docs/guide/quickstart#requirements

**Comment by DominiqueVial** [comment]
Update
GD is activated on the server but in fact is not activated! According to _Cpanel_ interface it's ok. Checking using `phpinfo()` show GD is not activated. You were right. My apologizes!

In fact no. GD is actived and works fine on webserver. And everything was fine before I installed this plugin.

I made changes on my website locally then I update the production file. Problem then occurs. Testing updates leeds to this error. Refreshing the page correct the problem. It seems related to time, maybe the plugin is called to early and is in competition with page construction.

> @DominiqueVial Hi, this is a problem of your webserver, since the PHP version installed was seemingly compiled without the GD library, which Kirby uses for image processing by default. You can find out more about Kirby’s requirements here: https://getkirby.com/docs/guide/quickstart#requirements
